x-pack/plugins/observability_solution/apm/public/components/app/mobile/service_overview/geo_map/index.tsx

```diff
@@ -27,7 +29,13 @@ export function GeoMap({
   dataView?: DataView;
 }) {
   const [selectedMap, selectMap] = useState(MapTypes.Http);
+  const { euiTheme } = useEuiTheme();
 
+  const styleColors = {
+    lineColor: euiTheme.colors.darkestShade,
+    labelColor: euiTheme.colors.darkestShade,
```

Comment:
Generally we don't want to add new usages of legacy shade tokens as they are deprecated and will be removed in the future.

Is `labelColor` a text color? If so, we could use `euiTheme.colors.textParagraph` here instead. Same then for `lineColor`.

For `emptyShade` we can use the semantic token `backgroundBasePlain` instead as there is no fitting border color.

Reply:
This works, and looks good, thanks for the feedback!